recipes/emsdk/config.yml

```
@@ -0,0 +1,3 @@
+versions:
+  "2.0.26":
```

The number of times I've made this mistake... just an insurmountable number of failures  :)

```suggestion
  "2.0.30":
```